dlp/snippets/src/test/java/dlp/snippets/DeIdentificationTests.java

```diff
@@ -555,6 +555,17 @@ public void testDeIdentifyWithDeterministicEncryption() throws IOException {
     assertThat(output).contains("Text after de-identification:");
   }
 
+  @Test
+  public void testReIdentifyWithDeterministicEncryption() throws IOException {
+    String textToReIdentify =
+        DeIdenitfyWithDeterministicEncryption.deIdentifyWithDeterministicEncryption(
+            PROJECT_ID, "My SSN is 372819127", WRAPPED_KEY, KMS_KEY_NAME);
+    ReidentifyWithDeterministicEncryption.reIdentifyWithDeterminsiticEncryption(
+        PROJECT_ID, textToReIdentify, WRAPPED_KEY, KMS_KEY_NAME);
+    String output = bout.toString();
+    assertThat(output).contains("Text after re-identification: ");
```

Comment:
it should contain the entire input that was provided in the deid method: "My SSN is 372819127"